src/opengl/ogl_bitmap.c

```diff
@@ -567,15 +567,21 @@ static void ogl_destroy_bitmap(ALLEGRO_BITMAP *bitmap)
 {
    ALLEGRO_BITMAP_EXTRA_OPENGL *ogl_bitmap = bitmap->extra;
    ALLEGRO_DISPLAY *disp;
+   ALLEGRO_DISPLAY *thisdisp;
```

Comment:
Would you mind naming this 'bmp_disp' to clarify what it is?

Reply:
will do - I remember thinking at the time that `thisdisp` wasn't a good name for it...